packages/aws-cdk-lib/aws-cognito/lib/user-pool.ts

```diff
@@ -767,6 +767,12 @@ export interface IUserPool extends IResource {
    */
   readonly userPoolArn: string;
 
+  /**
+   * User pool provider name
```

Comment:
Can we update the docstring to be more similar to some of the other attributes? Feels redundant to have an attribute `userPoolProviderName` with the description `User pool provider name`.